src/pages/workspace/accounting/qbd/advanced/QuickbooksDesktopAdvancedPage.tsx

```diff
@@ -50,9 +48,8 @@ function QuickbooksDesktopAdvancedPage({policy}: WithPolicyConnectionsProps) {
         <ConnectionLayout
             displayName={QuickbooksDesktopAdvancedPage.displayName}
             headerTitle="workspace.accounting.advanced"
-            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.CONTROL]}
```

Comment:
I do think so yes.

Otherwise it would be accessible on a Collect policy which is not what we want.

See:
- https://github.com/Expensify/App/blob/lucien/qbd-remove-beta/src/libs/PolicyUtils.ts#L373
- https://github.com/Expensify/App/blob/lucien/qbd-remove-beta/src/CONST.ts#L2258